### DISCUSSION TITLE
Dropdown hover only work for certain dropdown

### DISCUSSION THREAD

**Discussion by Tokioshi** [twbs/bootstrap#40744] [Q&A]
Hi, so I'm facing an issue where I had a navbar with dropdown on it. I want when user hover to the dropdown at the navbar, it will drop, just like how dropdown works. But, I had another dropdown that want to be different with the navbar dropdown. It will only work when user click it, doesn't hover it. But, my code made the dropdown that I don't want to drop when it's getting hover, become drop when hovered. Here is my code: [HTML](https://srcb.in/VcU8mae9Tx) and [CSS](https://srcb.in/dNJ2e4tWoe) And I've attach some screenshot of the website that might help.

[Navbar](https://cdn.discordapp.com/attachments/1274355662175997984/1274355662523990067/image.png?ex=66c34532&is=66c1f3b2&hm=643dd8d16921fc5a7b851ab6fc14246a2d5e72f76ba213be87f9dafd733c8b4c&)
[Other dropdown](https://cdn.discordapp.com/attachments/1274355662175997984/1274355662918520853/image.png?ex=66c34532&is=66c1f3b2&hm=64bb37eb13420c612672669c255c336e2405afad472b20d83d26cd564d70246b&)

**Comment by jonhubby** [comment]
The issue is that your hover styles are applied to all dropdowns instead of just the navbar ones. You could add a specific class to the navbar dropdown (e.g., .navbar-dropdown) and then target that in your CSS with something like .navbar-dropdown: hover. For the other dropdowns, you can just keep them tied to a: focus or onclick event.
